src/main/java/nl/tudelft/contextproject/model/Inventory.java

```diff
@@ -139,14 +153,15 @@ public Key getKey(ColorRGBA color) {
 					return key;
 				}
 			}
-	}
+		}
+
 		return null;
 	}
 	
 	/**
 	 * Gives the size of the current inventory.
 	 * @return
-	 * 		Current size
+	 * 		the current size of the inventory
 	 */
```

Comment:
I would suggest changing this to "the current amount of items in the inventory (bombs and keys)."